crates/autopilot/src/database/auction.rs

```diff
@@ -90,23 +95,21 @@ impl Postgres {
             .await?;
         let latest_settlement_block =
             database::orders::latest_settlement_block(&mut ex).await? as u64;
+        let quotes = {
+            let mut quotes = HashMap::new();
+            for order in &orders {
+                if let Some(quote) =
+                    database::orders::read_quote(&mut ex, &ByteArray(order.metadata.uid.0)).await?
+                {
+                    quotes.insert(order.metadata.uid, postgres::dto::into_domain(quote));
+                }
+            }
+            quotes
+        };
```

Comment:
If performance is of importance we can probably rewrite the solvable_orders query to also fetch the quotes in the same query.

Reply:
Would be nice to do this when solvable orders cache is refactored to work with `domain::Order` type directly.

Reply:
Implemented